src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -194,6 +194,7 @@ private void setUnderlineButton(Button button) {
     public void switchDashboard() {
         logger.info(String.format(CLICK_MODE_MESSAGE, "Dashboard"));
         switchMode(ModeEnum.DASHBOARD);
+        projectCompletionStatusPanel.updateOcp();
```

Comment:
can put this call under case DASHBOARD of switchMode(ModeEnum mode) method of MainWindow
(all switchMode inputs, whether via mouse input or CLI, go to this method)